docs/conceptual/array.contains['t]-function-[fsharp].md

```diff
@@ -0,0 +1,75 @@
+---
+title: Array.contains<'T> Function (F#)
+description: Array.contains<'T> Function (F#)
+keywords: visual f#, f#, functional programming
+author: liboz
+---
+
+# Array.contains<'T> Function (F#)
+
+Evaluates to `true` if the given element is in the given array.
```

Comment:
How about something like this:  `Evaluates to true if the given element is in the input array.`